### PR TITLE
adding in default overrides

### PR DESCRIPTION
**Pull request by adamdecaf**

### CODE REVIEW

**Comment by trentio** [comment]
:+1:  This should help with fetch not being able to pull dependencies ...

**Comment by adamdecaf** [comment]
I'm not sure what changed in versions with some dep, but this would lock us down so we're not trying to package jars with 1.6.x

**Comment by zmccoy** [comment]
I think this might have to do with banno-health having that override we added @adamdecaf.  https://github.com/Banno/banno-health/commit/84b4e1d2a9e1c0df2fcd3a036b1e014694619e3a#diff-fdc3abdfd754eeb24090dbd90aeec2ceR45

**Comment by adamdecaf** [comment]
So kafka-utils is adding 1.6.x to the classpath?

**Comment by rubbish** [comment]
kafka-utils is resolving 1.7.6

**Comment by rubbish** [comment]
I'm not sure this is totally necessary since slf4j-api is part of the standard deps: https://github.com/Banno/banno-sbt-plugin/blob/default-overrides/src/main/scala/deps.scala#L32

**Comment by rubbish** [comment]
> [debug] == resolving dependencies org.apache.zookeeper#zookeeper;3.4.6->org.slf4j#slf4j-api;1.6.1 [compile->master(*)]

**Comment by rubbish** [comment]
I did have to do this: https://github.com/Banno/hbase-persistence/commit/9c46ef66afd0510df725e733d5307a4d7177a4fb

**Comment by rubbish** [comment]
Another weird part to me is that it only does this on a release version and not a snapshot.

**Comment by adamdecaf** [comment]
Yea, I'm not sure why we haven't run into this on the usual snapshot builds.

File: src/main/scala/BannoIvy.scala [hunk 1]
```diff
@@ -38,11 +38,21 @@ object BannoIvy {
       "org.apache.avro" -> "avro-ipc",
       "org.apache.thrift" -> "thrift"
     )
+
+
+  val defaultOverrides =
+    Map[String, (String, String)]{
+      "1.7.6" -> ( "org.slf4j" -> "slf4j-api")
```
Comment (rubbish) [inline src/main/scala/BannoIvy.scala]: Can we make it so this depends on the version of slf4j defined in the `deps.scala`? I don't want another place to hunt down to change the version

**Comment by adamdecaf** [comment]
fixed

**Comment by adamdecaf** [comment]
retest this please

**Comment by adamdecaf** [comment]
I pushed what should be a fix.

**Pull request merged**